Library/Homebrew/formula.rb

```diff
@@ -1626,6 +1626,13 @@ def to_hash
         "head"   => head&.version&.to_s,
         "bottle" => !bottle_specification.checksums.empty?,
       },
+      "urls"                     => {
+        "stable" => {
+          "url"      => stable.url,
```

Comment:
Might be nice to do this inside `%w[stable devel].each do |spec_sym|` below where you already have a `spec` object. This will also handle the case when a `stable` spec doesn't exist.